hrm-service/package.json

```diff
@@ -13,7 +13,7 @@
     "test:unit": "jest unit-tests",
     "test:service": "cross-env POSTGRES_PORT=5433 run-s -c docker:compose:test:up test:service:migrate test:service:run docker:compose:test:down",
     "test:service:migrate": "node ./service-tests/db-init/migrate",
-    "test:service:run": "AWS_REGION=eu-west-1 SSM_ENDPOINT=http://localhost:4566 jest --verbose --maxWorkers=1 --forceExit service-tests",
+    "test:service:run": "cross-env AWS_REGION=eu-west-1 SSM_ENDPOINT=http://localhost:4566 jest --verbose --maxWorkers=1 --forceExit service-tests",
```

Comment:
Oops! Sorry about that. I have it fixed in flex-services as well while I try to get the serverless part of e2e tests working this morning.